src/Java.Interop/Java.Interop/JavaException.cs

```diff
@@ -22,7 +22,6 @@ unsafe public class JavaException : Exception, IJavaPeerable
 		JniObjectReferenceType  handle_type;
 	#pragma warning disable 0169
 		// Used by JavaInteropGCBridge
-		IntPtr                  weak_handle;
```

Comment:
You shouldn't remove this field *yet*, as it's still used by dotnet/android, even if it isn't "used":

  * https://github.com/dotnet/android/blob/852c38e0743e6e75981a491e7b430a5fe403027e/src/native/monodroid/monodroid-glue.cc#L766
  * https://github.com/dotnet/android/blob/852c38e0743e6e75981a491e7b430a5fe403027e/src/native/monodroid/monodroid-glue.cc#L849-L853
  * https://github.com/dotnet/android/blob/852c38e0743e6e75981a491e7b430a5fe403027e/src/native/monodroid/osbridge.hh#L83-L94
  * https://github.com/dotnet/android/blob/852c38e0743e6e75981a491e7b430a5fe403027e/src/native/monodroid/osbridge.cc#L25-L33

Removing it now will result in app startup aborts until dotnet/android is updated to no longer *mention* `weak_handle`.

Reply:
I didn't realize that all this code is duplicated in dotnet/android. I'll return back the field and try to see if we can remove it on dotnet/android side too. The lowest supported API level is 21 for .NET 9, so there doesn't seem to be a valid reason to keep this around.